s3/s3_transport/multipart_upload_libs3.json

```diff
@@ -0,0 +1,10 @@
+{
+    "keyfile" : "/projects/irods/vsphere-testing/externals/amazon_web_services-CI.keypair",
+    "hostname" : "s3.amazonaws.com",
+    "bucket_name": "justinkylejames1",
```

Comment:
consider... less justin?

Reply:
This file is being deleted.  It was used by another program before catch2 unit test cases existed.